pyuvdata/uvdata/src/miriad_wrap.pyx

```diff
@@ -157,12 +149,12 @@ cpdef hwrite_init(int item_hdl, str type) except +raise_miriad_error:
   return offset
 
 # hread_init surmises the type of a header item from the first few bytes
-cdef FIRSTINT(char s[4]):
+cdef int FIRSTINT(char s[4]):
   return (<int *>s)[0]
 
-cpdef hread_init(int item_hdl) except +raise_miriad_error:
+cpdef hread_init(int item_hdl) except +*:
```

Comment:
Ah yeah I have gone back and forth about this one myself. I think we can define `tuple`, I have done it before. However, I wonder if that actually provides any real help to the handoff since `tuple` is the python default return type like this anyway. I'll go ahead add it though just because we are pretty dogmatic about our typing in the rest of the code.

Reply:
Actually I just learned about this http://docs.cython.org/en/latest/src/userguide/language_basics.html#types

Reply:
So some trial and error seem to imply that ctuples do not behave nicely with strings/characters/etc. So I just set the output to `tuple` like you suggested.

Reply:
Ah, that makes sense that strings wouldn't behave. Thanks for doing the legwork, I think `tuple` is totally fine.